publisher.py

```diff
@@ -186,15 +200,13 @@ def parse_configuration(self):
             registry = Registry(config=config_reg)
             self.registries[registry.name] = registry
 
-        self.image_overrides = config['image_overrides']
         self.source_tag_tpl = config['source_tag']
         self.release_tags_tpl = config['release_tags']
         self.distribution = config['distribution']
 
-        if 'release_overrides' in config:
-            self.release_overrides = config['release_overrides']
-        else:
-            self.release_overrides = {}
+        self.retry_limit = config.get('retry_limit', 5)
```

Comment:
Put the default value in a single location.

Reply:
Done.